### PR TITLE
fix(core): overwrite title and aria-label in input-group component

### PR DESCRIPTION
**Pull request by OleksiyDrozdov**
#### Please provide a link to the associated issue.
Closes [#6220](https://github.com/SAP/fundamental-ngx/issues/6220) 

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [n/a] update `README.md`
- [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [n/a] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by Betrozov — APPROVED** [review]

**Review by platon-rov — CHANGES_REQUESTED** [review]

File: libs/core/src/lib/input-group/input-group.component.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
                 [compact]="compact"
                 [id]="_addOnButtonId"
                 [attr.tabindex]="buttonFocusable ? 0 : -1"
-                [attr.title]="iconTitle ? iconTitle : null"
+                [attr.title]="iconTitle? iconTitle: (glyphAriaLabel ? glyphAriaLabel: glyph)"
```
Comment (platon-rov) [inline libs/core/src/lib/input-group/input-group.component.html:33]: Could be simplified `iconTitle || glyphAriaLabel || glyph`.

File: libs/core/src/lib/input-group/input-group.component.html [hunk 2]
```diff
@@ -51,11 +51,11 @@
             [id]="_addOnNonButtonId"
             [compact]="compact"
             [placement]="placement"
-            [attr.aria-label]="glyphAriaLabel ? glyphAriaLabel : glyph ? glyph : addOnText"
-            [attr.title]="glyph ? glyph : addOnText"
+            [attr.aria-label]="glyphAriaLabel ? glyphAriaLabel : (glyph ? glyph : addOnText)"
+            [attr.title]="iconTitle? iconTitle: (glyph ? glyph : addOnText)"
```
Comment (platon-rov) [inline libs/core/src/lib/input-group/input-group.component.html:55]: Could be simplified `iconTitle || glyph || addOnText`.

**Review by platon-rov — CHANGES_REQUESTED** [review]

File: libs/core/src/lib/input-group/input-group.component.html (hunk 2 above)
Comment (OleksiyDrozdov) [reply libs/core/src/lib/input-group/input-group.component.html]: Done

File: libs/core/src/lib/input-group/input-group.component.html (hunk 1 above)
Comment (OleksiyDrozdov) [reply libs/core/src/lib/input-group/input-group.component.html]: Done

**Review by platon-rov — APPROVED** [review]

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
`fix(core): add title and aria-label` to which component? Please try to be more descriptive in your PR titles. This is what goes to the release notes. If you use a library and read in the release notes "add title and aria-label" does this bring any information to you?

**Review by InnaAtanasova — APPROVED** [review]

**Comment by puru-hk** [comment]
All issues are fixed.

**Review by puru-hk — APPROVED** [review]

**Review by mikerodonnell89 — APPROVED** [review]

**Pull request merged**